website/docs: add webfinger doc (#14099)
goauthentik/authentik: website/docs/add-secure-apps/providers/oauth2/webfinger_support.md

---
title: WebFinger support
---

## About WebFinger

> The WebFinger](https://webfinger.net/) protocol can be used to discover information about people or other entities on the Internet using standard HTTP methods. WebFinger discovers information for a URI that might not be usable as a locator otherwise, such as account or email URIs.

Reviewer: ```suggestion
The [WebFinger](https://webfinger.net/) protocol can be used to discover information about people or other entities on the Internet using standard HTTP methods. WebFinger discovers information for a URI that might not be usable as a locator otherwise, such as account or email URIs.
```

> The WebFinger](https://webfinger.net/) protocol can be used to discover information about people or other entities on the Internet using standard HTTP methods. WebFinger discovers information for a URI that might not be usable as a locator otherwise, such as account or email URIs.

Reviewer: other than that, suggested formulation:
```suggestion
The [WebFinger protocol](https://webfinger.net/) allows for the discovery of information about individuals or entities on the Internet through standard HTTP methods. It enables the retrieval of information associated with a URI that may not be directly usable as a locator, such as those for accounts or email addresses.
```

Reviewer: If I understood the protocol correctly

> ## Authentik WebFinger support

Reviewer: ```suggestion
## authentik WebFinger support
```

> Authentik provides a WebFinger endpoint when the **Default application** setting uses an OIDC provider. Instructions on how to set a **Default application** can be found in the [authentik Branding documentation](../../../sys-mgmt/brands.md#external-user-settings)

Reviewer: ```suggestion
authentik provides a WebFinger endpoint when the **Default application** setting uses an OIDC provider. Instructions on how to set a **Default application** can be found in the [authentik Branding documentation](../../../sys-mgmt/brands.md#external-user-settings)
```

---
title: WebFinger support
---

A [WebFinger](https://webfinger.net/) endpoint is available when the **Default application** setting uses an OIDC provider. To set a **Default application**:

> 1. Open the admin dashboard and navigate to **System** > **Brands**.

Reviewer: ```suggestion
1. Open the authentik Admin interface and navigate to **System** > **Brands**.
```
